pkg/http/request_test.go

```
@@ -732,6 +733,69 @@ func TestNewClientWithTracing_TimeoutConfiguration(t *testing.T) {
+		server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.WriteHeader(http.StatusOK)
+			_, _ = w.Write([]byte("this is a long response body that exceeds the limit"))
+		}))
+		defer server.Close()
+
+		client := NewClient(WithTracing(context.Background()), WithMaxResponseBytes(10))
+		resp, err := client.Get(server.URL)
+		if err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+		defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)